app/controllers/application_controller.rb

```diff
@@ -1,2 +1,16 @@
 class ApplicationController < ActionController::Base
+    #before_action :authorized
+    helper_method :current_myuser
+    helper_method :logged_in?
+    def current_myuser
+        Myuser.find_by(id: session[:myuser_id])
+    end
+
+    def logged_in?
+        !current_myuser.nil?
+    end
+
+    # def authorized
```

Comment:
Remove commented code if it is of no use

Reply:
> Which comment functionality is working ? From comments controller or the methods defined in events controller?
From controller it is working will remove the code from events controller.